ktor-client/ktor-client-plugins/ktor-client-resources/common/src/io/ktor/client/plugins/resources/builders.kt

```diff
@@ -0,0 +1,311 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.client.plugins.resources
```

Comment:
I would name the module `ktor-resources-client` and `ktor-resources-server` for consistency

Reply:
But then it will be inconsistant with other plugins. They all have names like `ktor-[client|server]-[pluginName]`